src/Examples/other_examples.jl

```diff
@@ -254,34 +254,6 @@ function fresnel(convex = true; kwargs...)
     Vis.drawtracerays(sys; test = true, trackallrays = true, numdivisions = 30, kwargs...)
 end
 
-function grating(; period = 1.0, θ = 0.0, λ = 0.55, kwargs...)
```

Comment:
These removals seem unrelated to the PR - is this intentional? If so maybe should be a separate PR

Reply:
It was intentional. A separate PR would be better in an ideal world but because it was a deletion of code that didn't work anymore it was simpler to add it into a single PR.